.github/workflows/maven.yaml

```diff
@@ -23,4 +23,4 @@ jobs:
           distribution: "adopt"
           cache: "maven"
       - name: Build with Maven
-        run: mvn -B package -T 2C --file pom.xml
+        run: mvn -B package --file pom.xml
```

Comment:
Compiling in parallel of ACL module is not supported on Windows?

Reply:
More failures are observed if parallel is enabled.